.gitignore

```diff
@@ -4,12 +4,13 @@ logs
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.idea
 
 # Node
 node_modules
 
 # Application
-data
+data*
```

Comment:
yea it ignores everything starting with data but you are right... might be overdoing it... (I made the nedb write to data2... so that is why I added the star... ~~was less work than adding a new line I guess... lazyness~~)